scripts/common/_common.sh

```diff
@@ -30,5 +30,12 @@ export HOST_DIR=$OUTPUT_ROOT/corehost
 [ -z "$CONFIGURATION" ] && export CONFIGURATION=Debug
 [ -z "$NOCACHE" ] && export NOCACHE=""
 
+[ -z "$BRANCH_TYPE"] && export BRANCH_TYPE=dev
```

Comment:
This looks good, but is "BRANCH_TYPE" environment variable the only way to specify this? Can we provide an option to build.sh?